submissions/usides/html-css-popup/css/main.css

```diff
@@ -50,29 +58,29 @@ nav.main-nav ul.nav-menu .direct-link {
   transition: all 0.2s;
 }
 
-nav.main-nav ul.nav-menu .direct-link:focus {
+.direct-link:focus {
   border-bottom: 3px solid #423e3e;
 }
 
-nav.main-nav ul.nav-menu .menu {
+.menu {
   position: relative;
   margin: 0 0.3rem;
 }
 
-nav.main-nav ul.nav-menu .menu input#btn-app {
+#btn-app {
```

Comment:
Styling by id is not a good practice.
Try to separate responsibilities:
- HTML tags - for markup
- CSS classes - for styling
- Id - for JS logic

Reply:
It's a recommendation for the future, it's not necessary to fix it.

Reply:
Done